wire-ios-request-strategy/Sources/Payloads/Processing/PayloadProcessing+Conversation.swift

```diff
@@ -501,3 +526,17 @@ extension Payload.ConversationEvent where T == Payload.UpdateConversationConnect
     }
 
 }
+
+private extension ZMConversation {
+
+    func firstSystemMessage(for systemMessageType: ZMSystemMessageType, in context: NSManagedObjectContext) -> ZMSystemMessage? {
```

Comment:
You don't need to pass in the context, each managed object already has a reference to a context.

Reply:
done